proposal/indented-syntax-improvements.md

```diff
@@ -0,0 +1,215 @@
+`()` must cause a statement break.
+
+`$var: 1 2\n3` and `$var: 1, 2\n, 3` can not be parsed to determine when the
+statement has ended. Alternates `$var: (1 2\n3)`, `$var: [1 2\n3]`, and `$var:
+(1, 2,\n 3)` can be parsed.
+
+Comma separated lists can not use a trailing comma to signify that a list will
+continue after the line break, as this would break existing stylesheets with
+trailing commas.
+
+Because arguments to functions and mixins are already wrapped in `()`, line
+breaks in arguments do not need to cause a statement break. Interpolations are
+wrapped in `#{}` so line breaks do not need to end statements.
+
+#### At-rules
+
+For any at-rule that is supported by native CSS, line breaks after the `@` and
+before a block or statement are not supported. This includes `@include`,
+(which overlaps with Sass), `@supports`, `@media`, `@keyframes` and any unknown
+at-rule.
+
+These rules should be emitted as is, with no special handling from Sass.
+
+### Design Decisions
+
+While some CSS at-rules may have contexts where a line break would not be
+meaningful, custom handling of line breaks is outside of the scope of this
+proposal. For instance, `@media (hover: hover) and \n (color)` is not supported,
+even though line breaks do not end statements after boolean operators in general
+SassScript.
+
+We considered borrowing alternate syntax from other languages, such as a leading
+`>` or `|` from YAML or a trailing `|` from HAML. These introduce syntax that is
+novel to Sass, and we instead opted to borrow syntax from the Scss format.
+
+These changes also allow authors using the indented syntax to add more explicit
+blocks with curly braces and line breaks with semicolons. While this introduces
+the changes for all authors, authors will still be able choose to limit the
+syntax with linters.
+
+In the indented syntax, authors can use a semicolon to explicitly end a
+statement. Subsequent lines in the same block still must have the same level of
+indentation. This means that the indented format won't support multiple
+statements on a single line, even if they are separated by a semicolon.
+
+```sass
+$font-stack: Helvetica, sans-serif;
+// or
+$primary-color: #333
+```
+
+In the indented syntax, authors can use curly braces to explicitly wrap a block
+of statements, but must use semicolons to separate statements inside that block.
+The braces essentially let authors opt in to Scss format for part of the
+document. We opted to require semicolons inside of curly braces to prevent
+issues with multiple nested formats.
+
+```sass
+$font-stack: Helvetica, sans-serif
+$primary-color: #333
+body {
+  font: 100% $font-stack;
+  color: $primary-color;
+}
+```
+
+When line breaks do not end statements, the line breaks are treated as white
+space, and do not have any requirements about the amount of indentation. The
+next statement's indentation is compared to the indentation at the start of the
+current statement to determine nesting or block closure.
```

Comment:
These feel like they belong outside of the Design Decisions section, since they're more documentation of the format than specifically descriptions of why we chose one option over another.